script/utils/curl.ts

```diff
@@ -0,0 +1,40 @@
+import axios from "axios";
+
+export async function getAccountDetail(accountId: string): Promise<any> {
+  return new Promise((resolve, reject) => {
+    axios
+      .get(`https://stacks-node-api.testnet.stacks.co/v2/accounts/${accountId}`)
+      .then((res) => {
+        resolve(res ? res.data: '');
+      })
+      .catch((error) => {
+        reject(error);
+      });
+  });
+}
+
+export async function getMicroBlock(): Promise<any> {
+  return new Promise((resolve, reject) => {
+    axios
+      .get('https://stacks-node-api.testnet.stacks.co/extended/v1/microblock/unanchored/txs')
+      .then((res) => {
+        resolve(res ? res.data: '');
+      })
+      .catch((error) => {
+        reject(error);
+      });
+  });
+}
+
+export async function getAnchorBlock(): Promise<any> {
+  return new Promise((resolve, reject) => {
+    axios
+      .get('https://stacks-node-api.testnet.stacks.co/extended/v1/tx?limit=200')
```

Comment:
A block may contain more than 200 transactions, there could be cases where you don't see the transaction because of this paging limit.

A better approach would be to record the current block_height before sending the tx, and then trying to retrieve the next blocks via `/extended/v1/block` until you see the tx_id listed

Reply:
Could you please elaborate a bit on how to do this, this is my first time taking this approach and I want to make sure I clearly understand the steps.